parameters:
 topic_name1: "poseupdate"
 buffer_size1: 1
 topic_name2: "scan"
 buffer_size2: 10

